behat.yml.dist

```diff
@@ -0,0 +1,7 @@
+default:
+    suites:
+        default:
```

Comment:
We can divide that for different suites and load only necessary context for suitable suite